broker-core/src/test/java/org/apache/qpid/server/store/BrokerStoreUpgraderAndRecovererTest.java

```diff
@@ -131,6 +132,7 @@ public void testUpgradeVirtualHostWithJDBCStoreAndBoneCPPool()
         context.put("qpid.jdbcstore.bigIntType", "mybigint");
         context.put("qpid.jdbcstore.varBinaryType", "myvarbinary");
         context.put("qpid.jdbcstore.blobType", "myblob");
+        context.put("qpid.jdbcstore.timestampType", "mytimestamp");
```

Comment:
Please undo the change as context variable "qpid.jdbcstore.timestampType" did not exist in previous model versions